### ISSUE TITLE
[Serious] Heading levels are out of order

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#54]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87494 |
| **Checkpoint** | 1.3 Adaptable › Headings (1.3.1.e) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Dashboard |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/dashboard> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/01/26 at 1:48 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

Heading levels are out of order in such a way that the structure of the content is not properly conveyed.
The heading structure of the page is not logical. There should be one heading level 1 that describes the purpose of the page, and subheading should decrease by 1 level.

## Recommendation

Rule : 
Headings SHOULD convey a clear and accurate structural outline of the sections of content of a web page.

How To Fix: 
Fix this issue by restructuring the heading outline to accurately reflect the hierarchical relationships of each section of content.

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/page-structure/headings/

Background: 
People who can see are able to quickly scan a page for headings and subheadings to understand the content and structure of a page. People who are blind do not have this ability if heading text is not marked in a way that screen readers can understand. Heading text and hierarchy must be identified semantically with heading markup. When heading markup is correctly applied, screen readers are able to: communicate that text is a heading and its level, provide a heading outline to users, and allow users to navigate directly from heading to heading.

## Source Code

```
<h1 class="h2">Recent Work Items</h1>
<h1 class="h2 is-flex is-align-items-center">
      <span class="material-icons mr-2" aria-hidden="true">notifications</span>
      Recent Notifications
    </h1>
<h1 class="h2">Deposits by Storage Option</h1>
<h5 class="text-label text-xs mb-2">Objects</h5>
<h5 class="text-label text-xs mb-2">Files</h5>
<h5 class="text-label text-xs mb-2">Events</h5>
```

## Screenshots

![dashboard_headings](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87494/dashboard_headings.png)
